streampipes-client-python/streampipes/model/common.py

```diff
@@ -30,6 +33,17 @@
 ]
 
 
+def random_letters(n: int):
```

Comment:
I'm not really happy with this function, but don't really have a better idea 🤔 
I'll try to come with one

Reply:
What don't you like about it?

Reply:
It's not about the implementation itself, it's more about that we only need it as default factory and does not really suit into the rest of the module 🤔